pandas/tests/sparse/test_frame.py

```diff
@@ -1118,6 +1120,58 @@ def test_isnotnull(self):
         tm.assert_frame_equal(res.to_dense(), exp)
 
 
+@pytest.fixture(params=['bsr', 'coo', 'csc', 'csr', 'dia', 'dok', 'lil'])
+def spmatrix(request):
+    tm._skip_if_no_scipy()
+    from scipy import sparse
+    yield getattr(sparse, request.param + '_matrix')
+
+
+@pytest.mark.parametrize('index', [None, list('abc')])
+@pytest.mark.parametrize('columns', [None, list('def')])
+@pytest.mark.parametrize('fill_value', [None, 0, np.nan])
+def test_from_to_scipy(spmatrix, index, columns, fill_value):
```

Comment:
so you can leave out ``float16`` we don't test that *anywhere* anyhow (and barely support it).

yeah the rest look fine (though your comparisons maybe slightly tricky as int is platform dependent).